pkgs/development/python-modules/nipype/default.nix

```diff
@@ -36,6 +36,10 @@ buildPythonPackage rec {
     sha256 = "47f62fda3d6b9a37aa407a6b78c80e91240aa71e61191ed00da68b02839fe258";
   };
 
+  postPatch = ''
+    substituteInPlace "nipype/info.py" --replace "PROV_VERSION = '1.5.0'" "PROV_VERSION = '1.5.2'"
```

Comment:
Can you remove this pinning instead? https://github.com/nipy/nipype/blob/master/nipype/info.py#L142
This only ask for trouble in future.

Reply:
do we really want to drop dependency constraints in upstream repos? I guess that this makes it easier to spot possible issues related to a hypotehtical bump of `pythonPackages.prov`.

Reply:
At least something like `< 1.6.0` or  `< 2`. Also see https://github.com/NixOS/nixpkgs/issues/34775#issue-295974989